spec/features/dams_collections_spec.rb

```diff
@@ -81,7 +81,40 @@
     visit dams_collections_path({:per_page=>100})
     expect(page).to have_content('Access: Curator Only')
   end
+end
+
+feature 'Visitor wants to see the search result page default sort is title after click "View Collection Items" url' do
+  before do
+    @unit = DamsUnit.create(pid: 'bb45454545')
+    @unit.damsMetadata.content = File.new('spec/fixtures/damsUnit.rdf.xml').read
+    @unit.save!  
+    @aCollection = DamsAssembledCollection.create(pid: "xx4473712z", visibility: "public", titleValue: "testCollection" )
+    @aCollection.save!
+    @testObj1 = DamsObject.create(pid: "xx2801340v")
```

Comment:
Only one other thought, related to these test objects and my other comment re: confirming the objects are sorted by title.

It seems like what we need these objects to have, as far as metadata, is a title. Anything else seems fairly irrelevant for the purpose of testing this new feature. I'm not sure how much difference in time it takes to persist and index these fixture files, but one (`soccomObject1`) appears to be a complex object. Could we instead just create two simple test objects and specify the minimal metadata we need to satisfy confirming that they appear ordered by title?

Something like:
```
...
@obj1 = DamsObject.create!(pid: "xx00000001", titleValue: "A Title", <unit and collection URI's>)
@obj2 = DamsObject.create!(pid: "xx00000002", titleValue: "B Title", <unit and collection URI's>)
..
```

Reply:
@mcritchlow After looking into it, I think it doesn't need to create or load new set of objects because there is existing loaded objects which is sufficient for testing this new feature. I've updated my tests. Please review. Thanks!